js/parts-more/PackedBubbleSeries.js

```diff
@@ -705,6 +707,35 @@ seriesType(
                 this.options.dataLabels.textPath = textPath;
             }
         },
+        /**
+         * The function responsible for calculating series bubble' s bBox.
+         * Needed because of exporting failure when useSimulation
+         * is set to false
+         * @private
+         */
+        seriesBox: function () {
+            var series = this,
+                data = series.data,
+                max = Math.max,
+                min = Math.min,
+                radius,
+                // bBox = [xMin, xMax, yMin, yMax]
+                bBox = [Infinity, -Infinity, Infinity, -Infinity];
```

Comment:
How about we start from `[plotLeft, plotLeft + plotWidth, plotTop, plotTop + plotHeight]` ?

Reply:
good idea, much more clear approach

Reply:
Done with e25a2ab